frontend/src/components/ui/Input/CommonInput/TextInput.jsx

```diff
@@ -0,0 +1,57 @@
+import SolidIcon from '@/_ui/Icon/SolidIcons';
+import React from 'react';
+import TrailingBtn from '../InputUtils/TrailingBtn';
+import { Input } from '../input';
+
+const TextInput = ({
+  size,
+  leadingIcon,
+  trailingAction,
+  trailingActionDisabled,
+  response,
+  disabled,
+  placeholder,
+  readOnly,
+  ...restProps
+}) => {
+  const inputStyle = `tw-border-border-default placeholder:tw-text-text-placeholder tw-font-normal disabled:tw-bg-[#CCD1D5]/30 ${
```

Comment:
is this correct `[#CCD1D5]/30`

Reply:
Yes according to tailwind documentation

refer: https://tailwindcss.com/docs/background-color#changing-the-opacity